### PR TITLE
Coverage: Use find and file module instead of shell command

### PR DESCRIPTION
**Pull request by akash2237778**
While understanding module encountered warning !!
Therefore Removed

Deprecated warning 

'use of command - shell: rm -f .coverage.*'

using Ansible module file, find

### CODE REVIEW

**Comment by akash2237778** [comment]
@tyll 
Sir,
Please Review changes
While understanding module I encountered warning !!
Therefore Removed Deprecated warning 'use of command - shell: rm -f .coverage.*' using Ansible module file, find

Thanks

File: tests/get-coverage.yml [hunk 1]
```diff
@@ -37,8 +37,18 @@
         state: absent
         path: "{{ coverage_file }}"
 
+    - name: find coverage file to delete
```
Comment (tyll) [inline tests/get-coverage.yml]:
```suggestion
    - name: find coverage files to delete
```

File: tests/get-coverage.yml [hunk 2]
```diff
@@ -37,8 +37,18 @@
         state: absent
```
Comment (tyll) [inline tests/get-coverage.yml]: Suggestion for the commit message:

```
coverage: Use find and file module instead of shell command
```

**Comment by tyll** [comment]
Thank you for adjusting the title of the PR. Please squash the commits together and adjust the commit message, too.

**Comment by akash2237778** [comment]
> Thank you for adjusting the title of the PR. Please squash the commits together and adjust the commit message, too.

Done Sir

**Comment by akash2237778** [comment]
Done sir
👍👍

**Comment by akash2237778** [comment]
Sir there is some trouble with the previous PR #177 due to update in master
repo

Therefore generated new PR with #180
Sorry for the inconvenience.

But now everything goes fine.

Thanks
Akash Saini

**Comment by akash2237778** [comment]
Sir any update on this?

**Comment by tyll** [comment]
> Sir there is some trouble with the previous PR #177 due to update in master repo Therefore generated new PR with #180 Sorry for the inconvenience. But now everything goes fine. Thanks Akash Saini

You need to learn how to update your branch without creating a new PR if you changed something. This might help: https://git-scm.com/book/en/v2/Git-Branching-Rebasing
By opening a new PR, the discussion from the old PR gets lost and the list of PRs becomes too long.

Closed as duplicate of #180

**Comment by akash2237778** [comment]
Sir,
This time updated my branch without creating a new PR. All thanks to docs
send by you.
Kindly run the checks and merge PR #180.
Thanks Sir

**Pull request closed without merging**